contracts/DomainController.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title DomainController
+ * @dev A simple domain registry contract that stores domain records associated with Ethereum addresses of the controllers.
+ * Each controller can create multiple domains. Domains are reserved forever.
+ */
+contract DomainController {
```

Comment:
// line 13 -16 over-logic 
// under requirement it is do not have any logic for storing data in struct 
// - 1 point